Weblate: Update translations

photoprismci opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/photoprism/photoprism?pullRequest=2122) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/photoprism/photoprism?pullRequest=2122) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/photoprism/photoprism?pullRequest=2122) it.</sub>

weblate committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 99.4% (549 of 552 strings)

Translation: PhotoPrism/Frontend
Translate-URL: https://translate.photoprism.app/projects/photoprism/frontend/zh_Hans/

weblate committed:
Added translation using Weblate (Turkish)

weblate committed:
Added translation using Weblate (Romanian)

weblate committed:
Translated using Weblate (Danish)

Currently translated at 95.8% (529 of 552 strings)

Translation: PhotoPrism/Frontend
Translate-URL: https://translate.photoprism.app/projects/photoprism/frontend/da/

weblate committed:
Added translation using Weblate (Bulgarian)
